include/fiction/io/read_sqd_layout.hpp

```diff
@@ -80,14 +80,24 @@ class read_sqd_layout_impl
                 throw sqd_parsing_error("Error parsing SQD file: no attribute 'type' in element 'layer'");
```

Comment:
I changed this file "read_sqd_layout.hpp", because I want to have siqad coordinates (x, y, z) as return type.

Reply:
Yes, that's correct!

Reply:
Fantastic!